client/src/Tools/_framework/Paths/Activities.tsx

```diff
@@ -314,7 +323,7 @@ export function Activities() {
               );
             }}
           >
-            Move Left
+            Move Up
```

Comment:
If should still be left if not list view, so something like `{listView ? "Up": "Left" }`. Same for Down/Right.